src/pathfora.js

```diff
@@ -142,7 +141,7 @@
   // NOTE HTML templates
   // FUTURE Move to separate files and concat
   /* eslint-disable indent */
-  var templates = {{templates}};
+  var templates = '{{templates}}';
```

Comment:
I see why you did this, because when I was doing my linting PR I did the same thing, but we decided against it. The problem is we don't really want to be finding/replacing our templates differently like this. Sadly there's no good way to disable this causing an error and @markhayden and I later agreed to only lint the `dist` version. I think if you run our lint task `gulp lint` as is, it will only check the dist file.

Idk if either of these are the best solution? Maybe it's something we should revisit? @markhayden any thoughts?